cluster/addons/fluentd-gcp/fluentd-gcp-configmap.yaml

```diff
@@ -397,13 +412,19 @@ data:
       # Use multiple threads for processing.
       num_threads 2
       use_grpc true
-      labels {
-        # The logging backend will take responsibility for double writing to
-        # the necessary resource types when this label is set.
-        "logging.googleapis.com/k8s_compatibility": "true"
-      }
+      # Use Metadata Agent to get monitored resource.
+      enable_metadata_agent true
     </match>
 
+    # Attach local_resource_id for 'k8s_node' monitored resource.
```

Comment:
NOTE: This entire `record_reformer` section sets up the `local_resource_id` for `k8s_node` resource. It's present in `fluentd-gcp-configmap.yaml` only (not `fluentd-gcp-configmap-old.yaml`).